aten/src/ATen/templates/RegisterDispatchKey.cpp

```diff
@@ -58,7 +67,15 @@ namespace {
 ${dispatch_anonymous_definitions}
 
 TORCH_LIBRARY_IMPL(aten, ${DispatchKey}, m) {
-  ${dispatch_registrations}
+  ${cpp_namespace}::Register${BackendName}${DispatchKey}Modules = [&]() {
+
+    ${dispatch_registrations}
+
+  };
+
+  #if EAGER_REGISTRATION
```

Comment:
ok- so i think what happens is

- for external backends, EAGER_REGISTRATION=1 causes the module registration function to be called at the same time it used to be called, in other words, at c++ static initialization time.
- for torchscript backend, EAGER_..=0 makes the c++ static initializer simply install a function pointer, and then only upon calling init_ts_backend is it finally invoked.

Could you write a comment to this effect in the template if so?